### ISSUE TITLE
New feature geometries tool does not appear correct

### ISSUE THREAD

**Issue by SPlanzer** [linz/gazetteer#156]
### My Confusion

After porting to QGIS3 the "New feature geometries tool" results in a dialog being shown once a new geom is drawn. 

This is shown as below:

![image](https://user-images.githubusercontent.com/10425696/88339646-847c2500-cd8e-11ea-8590-0a7302864572.png)
 
This is as expected as the original [source code](https://github.com/linz/gazetteer/blob/master/src/NZGBplugin/forms/featgeom.ui) has not been modified since 2015 and shows that the form shown is what was originally written. 

Note; this shows a QLineEdit as the dialogs main feature.

The test plan for porting I am working to, shows this should have a QLabel instead of the QLineEdit. This make more sense as at this stage in the workflow it seems the app should be just communicating with the user as opposed to fetching data. 

below is what the test plan indicates is expected after drawing a new geom
![image](https://user-images.githubusercontent.com/10425696/88340019-213ec280-cd8f-11ea-8b7d-2d7f75825901.png)

### For Now
I will continue with the test plan for now as what is most important is what the buttons connect to and the functionality they invoke. 

### Required
I will meet with @CMKMS-LINZ  to understand what is expected here and if I am missing something

**Comment by SPlanzer** [comment]
This form that is opening is a default QGIS layer form. 

Therefore the correct form is not being invoked and instead this default form

**Comment by SPlanzer** [comment]
The layer's editFormConfig points to the correct UI element (see below).

So why is it not using the correct form?

```
layer = QgsProject.instance().mapLayersByName('Gazetteer feature point')[0]
layer.editFormConfig().uiForm()


>>> '/home/splanzer/.local/share/QGIS/QGIS3/profiles/default/python/plugins/NZGBplugin/forms/featgeom.ui'
```

**Comment by SPlanzer** [comment]
Resolved by #156
